src/main/scala/djinni/CppGenerator.scala

```diff
@@ -68,6 +68,7 @@ class CppGenerator(spec: Spec) extends Generator(spec) {
     val underlyingType = if(e.flags) flagsType else enumType
 
     writeHppFile(ident, origin, refs.hpp, refs.hppFwds, w => {
+      writeDoc(w, doc)
```

Comment:
Without this change [this comment](https://github.com/jothepro/djinni-generator/blob/cppcli_include_idl_comments/src/it/resources/expected/my_enum/cpp-headers/my_enum.hpp#L8) would be missing.